shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -825,4 +837,31 @@ - (void)swipeWithEvent:(NSEvent*)event {
   // Not needed, it's handled by scrollWheel.
 }
 
+- (void)touchesBeganWithEvent:(NSEvent*)event {
+  if (@available(macOS 10.12, *)) {
+    NSTouch* touch = event.allTouches.anyObject;
```

Comment:
Can we make sure this contains either 0 or 1 touch? Maybe assert it?

Reply:
The number of touches shouldn't matter, though. If the user places two fingers simultaneously on the trackpad, we should still cancel inertia.